src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/ReviewUnusedParameters.cs

```diff
@@ -150,7 +151,8 @@ public override void Initialize(AnalysisContext context)
             if (eventsArgSymbol != null &&
                 method.Parameters.Length == 2 &&
                 method.Parameters[0].Type.SpecialType == SpecialType.System_Object &&
-                method.Parameters[1].Type.Inherits(eventsArgSymbol))
+                // UWP has specific EventArgs not inheriting from System.EventArgs. It was decided to go for a suffix match rather than a whitelist.
+                (method.Parameters[1].Type.Inherits(eventsArgSymbol) || method.Parameters[1].Type.Name.EndsWith("EventArgs", StringComparison.Ordinal)))
```

Comment:
I think you need to move the `eventsArgSymbol != null` check here:
```suggestion
                (eventsArgSymbol != null && method.Parameters[1].Type.Inherits(eventsArgSymbol) || method.Parameters[1].Type.Name.EndsWith("EventArgs", StringComparison.Ordinal)))
```

Reply:
Or you can just remove `eventsArgSymbol != null` completely given that Inherits extension method already does a null check.

Reply:
Good catch!